src/crt/cttz.src

```diff
@@ -0,0 +1,120 @@
+	assume	adl=1
+
+	section	.text
+	public	__bcttz
+__bcttz:
+	tst	a, 0Fh
+	jr	z, .high4
+	tst	a, 3
+	jr	z, .high6
+	cpl
+	and	a, 1
+	ret
+.high6:
+	dec	a
+	and	a, 6
+	ret	po
+	rrca
+	ret
+.high4:
+	tst	a, 030h
+	jr	z, .high2
+	dec	a
+	and	a, 014h
+	ret	po
+	ld	a, 5
+	ret
+.high2:
+	add	a, a
+	sbc	a, -8
+	ret	p
+	ld	a, 6
+	ret
+
+	section	.text
+	public	__scttz
+__scttz:
+	ld	a, l
+	or	a, a
```

Comment:
Matching `__icttz`'s first 3 bytes might compress better.

```suggestion
	xor	a, a
	or	a, l
```

Reply:
Good point, I always forget to think about compression alongside other size optimizations.